src/esm_calendar/esm_calendar.py

```diff
@@ -895,3 +895,38 @@ def sub_tuple(self, to_sub):
 
         new_date = self.from_list(result)
         return new_date
+
+    def is_in_interval(self, interval, sep="->") -> bool:
+        """
+        Finds if a date falls in an open interval.
+
+
+        Parameters
+        ----------
+        interval : str
+            A date interval string in which the 2 dates are separated by a ``sep``
+            string.
+        sep : str
```

Comment:
Just pass a tuple directly:


```suggestion
        interval : tuple of Date
            A date interval in which there are 2 Dates objects.
```

Reply:
thanks for the review, I'll take care of it later. I gave the interval as a string with a separator, because that's how it will be the same as the key in the `file` object. But I agree with you: let's keep this as independent as possible from `filedicts` and make the string transformations there